schemas/event.yml

```diff
@@ -153,16 +153,16 @@
       level: core
       type: date
       description: >
-        event.created contains the date when the event was created.
-
-        This timestamp is distinct from @timestamp in that @timestamp contains
-        the processed timestamp. For logs these two timestamps can be different
-        as the timestamp in the log line and when the event is read for example
-        by Filebeat are not identical. `@timestamp` must contain the timestamp
-        extracted from the log line, event.created when the log line is read.
-        The same could apply to package capturing where @timestamp contains the
-        timestamp extracted from the network package and event.created when the
-        event was created.
+        event.created contains the time when the event was first read by an
```

Comment:
"event.created contains the time when the event was first read by an" should be
"event.created typically contains the date/time when the event was first read by an"